### PR TITLE
Doc updates for syncing

### PR DESCRIPTION
**Pull request by Simran-B**
- Update doc links
- Fix some capitalization in docs
- Fix a typo

### CODE REVIEW

File: docs/Drivers/JS/Reference/Collection/Indexes.md [hunk 1]
```diff
@@ -242,6 +243,12 @@ assert.equal(result.id, index.id);
 
 const result = await collection.index(index.id.split("/")[1]);
 assert.equal(result.id, index.id);
+
+// -- or --
+
+const result = await collection.index(index.name);
+assert.equal(result.id, index.id);
+assert.equal(result.name, index.name);
```
Comment (pluma) [inline docs/Drivers/JS/Reference/Collection/Indexes.md]: Can you split these additions out into a separate PR? The URL changes will likely cause a massive merge conflict with the next-major branch.

Comment (Simran-B) [reply docs/Drivers/JS/Reference/Collection/Indexes.md]: Reverted. The changes are in docs repo. This was only to reduce the amount of changes that show up in manual three-way merging when syncing the external repos into the docs repo.

File: docs/Drivers/JS/Reference/Analyzer.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Analyzer API
 
 These functions implement the
-[HTTP API for manipulating analyzers](https://docs.arangodb.com/latest/HTTP/Analyzers.html).
+[HTTP API for manipulating Analyzers](https://www.arangodb.com/docs/stable/http/analyzers.html).
```
Comment (pluma) [inline docs/Drivers/JS/Reference/Analyzer.md]: Is this a trademark/brand thing? I don't think we capitalise View or Collection except when talking about types, so it seems odd to always capitalise Analyzer.

Comment (Simran-B) [reply docs/Drivers/JS/Reference/Analyzer.md]: Not a brand in that sense, but I changed the capitalization of Analyzers and Views throughout the main docs in the ArangoSearch refactor. The View concept is quite unique compared to SQL Views, and we also do this for Agents, DBServers and Coordinators. I agree that it is a bit odd to not do this for collections at the same time, but on the other hand it is a common term among NoSQL DBMS and our regular document collections aren't much idiosyncratic. In general and for the future, I would like to capitalize our terminology consistently and add links to the glossary where it makes sense.

Comment (pluma) [reply docs/Drivers/JS/Reference/Analyzer.md]: Okay, feels weird to arbitrarily capitalize some ArangoDB-related terms but fair enough as long as it's consistent.
